app/ide-desktop/lib/dashboard/src/providers/BackendProvider.tsx

```diff
@@ -70,14 +69,20 @@ export default function BackendProvider(props: BackendProviderProps) {
   )
 }
 
+function useBackendContext() {
```

Comment:
The name should probably contain something like `Strict`, e.g. `useStrictBackendContext()` to mark that this function will fail if the backend is not provided.